components/editor/modules/document/plain.js

```diff
@@ -121,10 +129,12 @@ export default ({ rule, subModules, TYPE }) => {
     rules: [documentRule]
   })
 
-  const newDocument = ({ title, template }, me) =>
+  const newDocument = ({ title, schema }, me) =>
     serializer.deserialize(
       parse(`---
-template: ${template}
+template: ${schema}
+title: ${titleModule ? null : title}
```

Comment:
let's use `jsYaml` (already an dependency) to prevent encoding issues and null title data in newsletters:
```js
import jsYaml from 'js-yaml'
jsYaml.safeDump({template: schema, ...(titleModule && { title }), auto: true})
```